modules/installation-complete-user-infra.adoc

```diff
@@ -133,6 +134,7 @@ The command succeeds when the Cluster Version Operator finishes deploying the
 ====
 
 . Confirm that the Kubernetes API server is communicating with the pods.
++
 .. To view a list of all pods, use the following command:
```

Comment:
Do we need a `+` between a step and a substep? I don't think it's wrong, just curious if it is needed (or will be needed).